pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -478,6 +478,11 @@ type TiKVSpec struct {
 	// EnableNamedStatusPort enables status port(20180) in the Pod spec.
 	// If you set it to `true` for an existing cluster, the TiKV cluster will be rolling updated.
 	EnableNamedStatusPort bool `json:"enableNamedStatusPort,omitempty"`
+
+	// MaxScaleInReplicas configures max scale in replicas for TiKV stores.
+	// Defaults to 1
+	// +optional
+	MaxScaleInReplicas *int32 `json:"maxScaleInReplicas,omitempty"`
```

Comment:
what about use `ScaleInParallelism`

Reply:
Sounds good. Already done. (Sorry for the slow response.